config/configload/merge.go

```diff
@@ -214,6 +214,21 @@ func mergeServers(bodies []*hclsyntax.Body, proxies map[string]*hclsyntax.Block)
 								return nil, newMergeError(errUniqueLabels, subBlock)
 							}
 
+							if attr, ok := subBlock.Body.Attributes[proxy]; ok {
```

Comment:
extract this to a function to be called for both free and api endpoints

Reply:
Done